src/components/Exchange/PositionSeller.js

```diff
@@ -74,6 +88,30 @@ function getTokenAmount(usdAmount, tokenAddress, max, infoTokens) {
   return usdAmount.mul(expandDecimals(1, info.decimals)).div(max ? info.minPrice : info.maxPrice);
 }
 
+function getSwapLimits(infoTokens, fromTokenAddress, toTokenAddress) {
+  const fromInfo =  getTokenInfo(infoTokens, fromTokenAddress)
+  const toInfo = getTokenInfo(infoTokens, toTokenAddress)
+
+  const maxInUsd = fromInfo.maxUsdgAmount
+    ?.sub(fromInfo.usdgAmount)
+      .mul(expandDecimals(1, USD_DECIMALS))
+      .div(expandDecimals(1, USDG_DECIMALS));
+
+  const maxIn = maxInUsd?.mul(expandDecimals(1, fromInfo.decimals)).div(fromInfo.maxPrice).toString();
+  const maxOut = toInfo.availableAmount?.gt(toInfo.poolAmount?.sub(toInfo.bufferAmount))
```

Comment:
`gt()` would fail if you pass undefined to it. i.e. in case availableAmount != undefined, but toInfo.poolAmount is undefined it will raise error